src/Sylius/Bundle/FlowBundle/Process/Builder/ProcessBuilder.php

```diff
@@ -79,6 +79,10 @@ public function add($name, $step)
             throw new \InvalidArgumentException('Step added via builder must implement "Sylius\Bundle\FlowBundle\Process\Step\StepInterface"');
         }
 
+        if (false === $step->isActive()) {
```

Comment:
I think should = must. If it is returning anything else then it is behaving badly. I really prefer the strict check.